config/manager/manager.yaml

```diff
@@ -66,6 +66,7 @@ spec:
           - "--upstream=http://127.0.0.1:8080/"
           - "--logtostderr=true"
           - "--v=10"
+          - "--http2-disable=true"
```

Comment:
Doesn't seem to match with CLI arg name

Reply:
that's because this is how the argument is called in kube-rbac-proxy for which this argument is for.